crypt-static.c

```diff
@@ -17,6 +18,13 @@
 #include "crypt-port.h"
 #include "xcrypt.h"
 
+#if ENABLE_OBSOLETE_API_ENOSYS
+char *
+fcrypt (const char *key, const char *setting);
```

Comment:
Why is it necessary to add a prototype for fcrypt?  Also, putting the function name on the line after the return type is only for function _definitions_ (the point of that convention is you can do `grep '^function_name' *.c` to find only the definition).  Also, the inclusion of errno.h should come first.

Reply:
> Why is it necessary to add a prototype for fcrypt?

We compile with `-Wmissing-prototypes -Werror` by default.  For that reason we need to prototype `fcrypt` as soon as we don't declare it a `strong_alias` for `crypt`.


> Also, putting the function name on the line after the return type is only for function _definitions_ (the point of that convention is you can do `grep '^function_name' *.c` to find only the definition). Also, the inclusion of errno.h should come first.

Fixed.

Reply:
I've moved the prototype for `fcrypt` with a conditional into `crypt-port.h`.